pkg/perf/images/create_images.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+)
+
+func main() {
+	for i, imageName := range imageNameList {
+		if err := makeImageDir(imageName); err != nil {
+			log.Fatal(err)
+		}
+
+		createLayout(imageName)
+		manifestName, manifestSize := createBlobs(i*size, imageName)
+		createIndex(imageName, manifestName, manifestSize)
+	}
+}
+
+func createLayout(imageName string) {
```

Comment:
Per our offline discussion, please try to use the github.com/opencontainers/image-spec/specs-go/v1 and the github.com/opencontainers/umoci libraries. A lot of code in this PR is duplicating that functionality.